R/mapDGEobj.R

```diff
@@ -107,9 +122,11 @@ mapDGEobj <- function(dgeObj,
 
         mapDGEplot <- canvasXpress::canvasXpress(data              = list(nodeData = nodes, edgeData = edges),
```

Comment:
No need for **mapDGEplot** mapDGEplot  return the plot function output for cx and igraph directly